crates/driver/src/infra/api/routes/reveal/dto/solution.rs

```diff
@@ -2,12 +2,11 @@ use {serde::Deserialize, serde_with::serde_as};
 
 #[serde_as]
 #[derive(Debug, Deserialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
 pub struct Solution {
     /// Unique ID of the solution (per driver competition), to reveal.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     pub solution_id: u64,
     /// Auction ID in which the specified solution ID is competing.
-    #[serde_as(as = "serde_with::DisplayFromStr")]
-    pub auction_id: i64,
+    pub auction_id: Option<String>,
```

Comment:
We can keep `deny_unknown_fields` by adding `#[serde(default)]` for the `auction_id` field.

Reply:
Also, why `Option<String>`?
`#[serde_as(as = "Option<serde_with::DisplayFromStr>")]` should help to make it Option<i64>.

Reply:
> #[serde_as(as = "Option<serde_with::DisplayFromStr>")]

Good point, I didn't figured it out that it can be used like this. I will update that.